src/components/SlateEditor/plugins/related/EditRelated.jsx

```diff
@@ -183,6 +184,24 @@ class EditRelated extends React.PureComponent {
                                       providedInner.dragHandleProps
                                     }
                                     key={article.id}>
+                                    {article.id ===
+                                      'external-learning-resources' && (
+                                      <Button
+                                        stripped
+                                        data-testid="showAddExternal"
+                                        onClick={() => {
+                                          this.setState({
+                                            index,
+                                            url: article.url,
+                                            title: article.title,
+                                          });
+                                          this.toggleAddExternal();
+                                        }}>
+                                        {t(
+                                          'form.content.relatedArticle.changeExternal',
```

Comment:
Kanskje bruke dette som tooltip på en knapp med blyantikon?
![image](https://user-images.githubusercontent.com/8956884/104551386-16c2fe80-5637-11eb-8486-aa324b23f168.png)
Og plassere blyantikonet etter tittelen eller ved siden av krysset for å fjerne.

Reply:
Grunnen til at jeg har plassert den der jeg har er fordi strukturen til "RelatedArticle" er i ndla/ui

Reply:
Åja. Skjønner.

Reply:
Jeg gjorde en liten update på hvordan det så ut nå, du får se om det er bedre

Enig med at det er rare navn på knapper, har bare tatt de som alt var der.